ui/Dockerfile-dev

```diff
@@ -0,0 +1,13 @@
+FROM node
```

Comment:
Add to top of Dockerfile:
```
# Production deployment
# App Engine requires "Dockerfile", so this can't be named "Dockerfile.prod".
```

Rename this to Dockerfile.local to be consistent with JM.

Remove "# local environment" comment from this file, it's obvious from "Dockerfile.local"

Reply:
Done